docs/quickstart/quickstart-docker.rst

```diff
@@ -69,7 +69,7 @@ Start KSQL
 
    .. code:: bash
 
-       $ docker-compose exec ksql-cli ksql-cli local --bootstrap-server kafka:29092
+       $ docker-compose exec ksql-cli ksql http://ksql-server:8080
```

Comment:
@dguy : Shouldn't `ksql-cli` be called `ksql` by now?

Reply:
`ksql-cli` is the name of the docker container for the cli. can change it if you like

Reply:
Ah sorry, misread the order of arguments.  Sorry!